### ISSUE TITLE
Implement K-Nearest-Neighbors on Bollinger Bands

### ISSUE THREAD

**Issue by bbell03** [bvkin/trade_bot#42]
Start learning on trading data and strategy for enhanced decision making.

**Comment by bbell03** [comment]
`import pandas as pd
from sklearn.neighbors import KNeighborsClassifier
from sklearn.model_selection import train_test_split

X, y = get_clean_Xy(data)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=.5, random_state=0)

clf = KNeighborsClassifier(7)  # Model the output based on 7 "nearest" examples
clf.fit(X_train, y_train)

y_pred = clf.predict(X_test)

_ = pd.DataFrame({'y_true': y_test, 'y_pred': y_pred}).plot(figsize=(15, 2), alpha=.7)
print('Classification accuracy: ', np.mean(y_test == y_pred))`

**Comment by bbell03** [comment]
https://colab.research.google.com/github/kernc/backtesting.py/blob/master/doc/examples/Trading%20with%20Machine%20Learning.ipynb#scrollTo=HjgBAM5CgJv8
